hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ResourceAttributeEnricher.java

```diff
@@ -60,6 +66,9 @@ public void enrichEvent(StructuredTrace trace, Event event) {
                             attributeValue
                                 .getValue()
                                 .substring(attributeValue.getValue().lastIndexOf('/') + 1));
+                      } else if (resourceAttributeKey.equals(DEPLOYMENT_TYPE_KEY)) {
```

Comment:
I recommend this be converted to a CASE statement, as more such conditions can be added in future.